OpenRA.Mods.Common/Widgets/Logic/Editor/SaveMapLogic.cs

```diff
@@ -28,6 +28,20 @@ struct MapFileTypeInfo
 			public string UiLabel;
 		}
 
+		class SaveDirectory
+		{
+			public readonly Folder Folder;
+			public readonly string DisplayName;
+			public readonly MapClassification Classification;
+
+			public SaveDirectory(Folder folder, MapClassification classification)
+			{
+				Folder = folder;
+				DisplayName = Platform.UnresolvePath(Folder.Name);
```

Comment:
Can we use the classification instead here? The average user won't have any idea what the caret stands for, for example, or where './mods/ra/maps' is, so those paths don't convey any real information.